packages/vaex-ml/vaex/ml/.coverage

```diff
@@ -0,0 +1 @@
+!coverage.py: This is a private format, don't read it directly!{"lines":{}}
```

Comment:
I don't think this file should go into the repo.

Reply:
Agreed! I tried to exclude it.. i will check the gitignore file again

Reply:
Same for the pycache/ like things.